plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2425,6 +2432,9 @@ public ConnectorTableHandle beginDelete(ConnectorSession session, ConnectorTable
         if (retryMode != NO_RETRIES) {
             throw new TrinoException(NOT_SUPPORTED, "Deleting from Hive tables is not supported with query retries enabled");
         }
+        if (isSparkBucketedTable(table)) {
+            throw new TrinoException(NOT_SUPPORTED, format("Deleting from Spark bucketed table %s is not supported", tableName));
```

Comment:
nit: the message above says "tables" while here we say "table". 
Since it's not ambiguous which table insert, update, delete operate on, i'm leaning towards "... from Spark bucketed tables is not supported" wording.
this has additional advantage, that this isn't table-specific, but applies to all spark bucketed tables

 (here & above)